src/main/java/hng_java_boilerplate/config/DatabaseSeeder.java

```diff
@@ -1,124 +1,124 @@
-package hng_java_boilerplate.config;
```

Comment:
Why did you comment out the database seeder?

Reply:
because my update to the user class would affect it since I added more field and that would require updating the database seeder class. Do I update the database seeder class to accomodate my changes?